### ISSUE TITLE
About 3D simulation.

### ISSUE THREAD

**Issue by parksanha** [eclipse/sumo#8829]
Is there any tutorial that tells me sumo 3d simulation?

**Comment by palvarezlopez** [comment]
Hi @parksanha . What do you need exactly?

**Comment by parksanha** [comment]
Sorry for not being specific.
I want to convert a 2d simulation to a 3d simulation.

I checked the link below, but I don't know how to use it.
Can you help with this?
https://sumo.dlr.de/docs/sumo-gui.html#3d_visualization

**Comment by palvarezlopez** [comment]
Currently the 3D simulation is very experimental, but you can activate it using 3D button in SUMO-GUI
![3D](https://user-images.githubusercontent.com/8238561/125759929-34a1ed42-9b50-4e72-afe1-becf958196fa.png)

For more question please use our mailing list

**Comment by Laudarisd** [comment]
Hi @palvarezlopez I tried to follow your instruction to visualize my 2D simulation in 3D.I was able to visualize street but couldn't get any other background information. So is there anything that I am missing?

Actually I want to do following in 3D simulation.
1) Transform image background in to 3D (just 3D shape is okay).


 And when we change vehicle shape into box it doesn't show real shape. Is it possible to fix it?

 Here is an image from my simulation(2D and 3D visualization)
![image](https://user-images.githubusercontent.com/53330430/126578889-10ed23cc-8c10-4bc7-9338-c4326bcf6b6e.png)
